core/settingscog.py

```diff
@@ -73,6 +81,10 @@ async def settings_handler(self, ctx,
                 settings.update(guild, 'default_steps', set_maxsteps)
                 reply = reply + '\nDefault steps value is too high! Lowering to ' + str(set_maxsteps) + '.'
 
+        if set_count != 1:
```

Comment:
This line here implies that it can't be set back to 1 once changed

Reply:
That appears to be the case. I changed count to 2 and it won't go back to 1.
The settings summary becomes confused too
![image](https://user-images.githubusercontent.com/2993060/198904204-0b683896-a3dc-4c23-a0db-ad04dcc42e90.png)